docs/source/markdown/podman-create.1.md

```diff
@@ -497,10 +497,16 @@ To specify multiple static IPv6 addresses per container, set multiple networks u
 
 #### **--ipc**=*ipc*
 
-Default is to create a private IPC namespace (POSIX SysV IPC) for the container
-		`container:<name|id>`: reuses another container shared memory, semaphores and message queues
-		`host`: use the host shared memory,semaphores and message queues inside the container. Note: the host mode gives the container full access to local shared memory and is therefore considered insecure.
-		`ns:<path>` path to an IPC namespace to join.
+Set the IPC namespace mode for a container. The default is to create
+a private IPC namespace.
+
+- "": Use Podman's default, defined in containers.conf.
+- **container:**_id_: reuses another container shared memory, semaphores and message queues
+- **host**: use the host shared memory,semaphores and message queues inside the container. Note: the host mode gives the container full access to local shared memory and is therefore considered insecure.
```

Comment:
```suggestion
- **host**: use the host's shared memory, semaphores, and message queues inside the container. Note: the host mode gives the container full access to local shared memory and is therefore considered insecure.
```

Reply:
on the fence.  Should we bold the word "Note" here?  Just to draw the eye to the potential security issue.